src/Microsoft.AspNetCore.Razor.Evolution/Legacy/TagHelperDirectiveSpanVisitor.cs

```diff
@@ -1,170 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-
-namespace Microsoft.AspNetCore.Razor.Evolution.Legacy
-{
-    internal class TagHelperDirectiveSpanVisitor
```

Comment:
This code was doing things the hard way, I had to touch it so I replaced it with something much simpler.